website/gatsby-config.js

```diff
@@ -45,7 +45,20 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-google-fonts',
       options: {
-        fonts: ['roboto mono'],
+        fonts: [
+          'roboto mono',
+          // https://design.google/library/spectral-new-screen-first-typeface/
+          // 'Spectral:400,700',
+          // https://design.google/library/choosing-web-fonts-beginners-guide/
+          // 'Libre+Baskerville:400,400i,700',
+          'Raleway:400,400i,700,700i',
```

Comment:
This is used for the typography content.

Reply:
/cc @celeste-horgan

Reply:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/1110551/56850112-2435d380-68fe-11e9-8083-41509aeeec57.png">

Reply:
Nice!

For the docs site, I used https://grtcalculator.com/ with a Open Sans at base size of 16px and body content width @ 800px. 

Perhaps we should talk to the designers about using a consistent typography scale? Sven is working on some stuff for the Import API docs for us right now.

Reply:
FYI we do use REM in UI-Kit, so it respects the base HTML font size...